geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -452,21 +456,39 @@ public static InternalDistributedSystem newInstance(Properties config,
   }
 
   /**
-   * creates a non-functional instance for testing
+   * Creates a non-functional instance for testing.
    *
-   * @param nonDefault - non-default distributed system properties
+   * @param distributionManager the distribution manager for the test instance
+   * @param properties properties to configure the test instance
    */
-  public static InternalDistributedSystem newInstanceForTesting(DistributionManager dm,
-      Properties nonDefault) {
-    InternalDistributedSystem sys = new InternalDistributedSystem(nonDefault);
-    sys.config = new RuntimeDistributionConfigImpl(sys);
-    sys.dm = dm;
-    sys.isConnected = true;
-    return sys;
+  public static InternalDistributedSystem newInstanceForTesting(
+      DistributionManager distributionManager, Properties properties) {
+    DistributionConfig config = new DistributionConfigImpl(properties);
```

Comment:
This variable is never used and didn't seem to affect `./gradlew build`

Reply:
Good catch!